cypress/integration/visitorCanSeeMultipleLanguages.spec.js

```diff
@@ -0,0 +1,25 @@
+describe("Visitor can change language from", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v0/articles",
+      response: "fixture:articles.json"
+    });
+    cy.visit("http://localhost:3001");
+    cy.get('#english').click({ force: true });
```

Comment:
Why do you need to force this click? (force means that you click on the element even if it is hidden)

Reply:
<img width="559" alt="Screenshot 2019-08-27 at 4 07 22 PM" src="https://user-images.githubusercontent.com/51204927/63778276-ca154100-c8e4-11e9-96e2-7b4c40056398.png">

Reply:
The select language menu is not a dropdown, but rather a hidden menu item so thats why we need force click

Reply:
Well, then you should click on the dropdown first and then click on the language.

Reply:
The particular Semantic UI dropdown menu we are using does not respond to a click action, it shows the hidden selection upon mouse hover.

Cypress specifically does not support "on hover" testing.
https://docs.cypress.io/api/commands/hover.html#Workarounds

The cypress work around to mimicking hover is to invoke the JS that causes the pull down items to show.  However, this particular "simple" dropdown menu does not invoke JS.
https://react.semantic-ui.com/modules/dropdown/#types-simple

Using the click({ force: true}) method is the most straight forward testing solution with our current menu configuration.

Reply:
Okay. Thank you. I pulled down the code and you are totally right. I tried a variety of methods (like `trigger('mouseenter')` or `mouseover` or `mousedown`), but I could not get the dropdown to open. We will go with your solution despite its flaws.